scripts/add_webhook.sh

```diff
@@ -16,19 +16,24 @@ if [ -z "${GITHUB_SECRET_TOKEN}" ]; then
   exit 1
 fi
 
+if [ -z "${GITHUB_AUTH_TOKEN}" ]; then
```

Comment:
@ehelms I guess it could be renamed to that - although in this case I believe it's a different token than the one used to update labels (different scopes needed)